_opensearch/cluster-manager-task-throttling.md

```diff
@@ -0,0 +1,104 @@
+---
+layout: default
+title: Cluster manager task throttling
+nav_order: 70
+has_children: false
+---
+
+# Cluster manager task throttling
+
+For many cluster state updates, such as defining a mapping or creating an index, nodes submit tasks to the cluster manager. The cluster manager maintains a pending task queue for these tasks and runs them in a single-threaded environment. When nodes send tens of thousands of resource-intensive tasks, like `put-mapping` or snapshot tasks, these tasks pile up in the queue, and the cluster manager is flooded. This affects the cluster manager performance, and may in turn affect the availability of the whole cluster. 
```

Comment:
suggestion only:
"When nodes send tens of thousands of resource-intensive tasks, like `put-mapping` or snapshot tasks, these tasks can pile up in the queue and flood the cluster manager." 
"This affects cluster manager performance..."
or
"This affects the cluster manager's performance..."

Reply:
This is good. I'll change. Thanks!